edb/schema/reflection/reader.py

```diff
@@ -100,6 +100,9 @@ def parse_into(
         name = s_name.name_from_string(entry['name__internal'])
         layout = schema_class_layout[mcls]
 
+        if base_schema.has_object(objid) and str(name) != '__schema_version__':
```

Comment:
Why `__schema_version` specifically?  Is there a more general rule?

Reply:
I made it handle the Version types more generally. I don't know if there's something more general than that

Reply:
But it is because `__schema_version__` is special and get updated despite also being in the base schema